src/components/Panel.ts

```diff
@@ -434,7 +434,7 @@ class Panel implements FlickingPanel {
   public removeElement(): void {
     if (!this.viewport.options.renderExternal) {
       const element = this.element;
-      element.parentNode!.removeChild(element);
+      element.parentNode && element.parentNode.removeChild(element);
```

Comment:
The old code also looks good, why did it change? :)

Reply:
There can be a case that elements can be removed without parentNode.
It's when `remove()` is called by that removed panel is part of visible panels so that it could be removed twice. That code prevents such case.